doc/classes/AABB.xml

```diff
@@ -188,15 +188,15 @@
 			<param index="0" name="from" type="Vector3" />
 			<param index="1" name="dir" type="Vector3" />
 			<description>
-				Returns [code]true[/code] if the given ray intersects with this [AABB]. Ray length is infinite.
+				Returns the point of intersection of the given ray with this [AABB] or null if there is no intersection. Ray length is infinite.
```

Comment:
```suggestion
				Returns the point of intersection of the given ray with this [AABB] or [code]null[/code] if there is no intersection. Ray length is infinite.
```

Same for the other line.

*Note:* Please make modifications by _amending_ the commit, and *force pushing* the changes, so that it stays as a single commit (see [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request)).

Reply:
Thanks, solved.